bindsnet/network/nodes.py

```diff
@@ -181,14 +181,14 @@ def step(self, inpts, dt):
         self.refrac_count[self.refrac_count != 0] -= dt
 
         # Check for spiking neurons.
-        self.s = (self.v >= self.thresh) & (self.refrac_count == 0)
+        self.s = (self.v >= self.thresh)
 
         # Refractoriness and voltage reset.
         self.refrac_count.masked_fill_(self.s, self.refrac)
         self.v.masked_fill_(self.s, self.reset)
 
         # Integrate input and decay voltages.
-        self.v += inpts
+        self.v += (self.refrac_count  == 0).float() * inpts
```

Comment:
There's an extra space here: `(self.refrac_count  == 0).float() *` should be `(self.refrac_count == 0).float() *`

Reply:
I'm sorry, I dont see the different between the two snippets and I dont see the extra space

Reply:
In your code, there's an extra space between `self.refrac_count` and the `==` operator.